<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpenFileDialog</class>
 <widget class="QDialog" name="OpenFileDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>629</width>
    <height>383</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Show file</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Show graph from saved log files.&lt;/p&gt;&lt;p&gt;Please select log file to show.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>&amp;Acceleration</string>
     </property>
     <property name="buddy">
      <cstring>accButton</cstring>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLineEdit" name="accFileEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="accButton">
       <property name="text">
        <string>Select a file</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Angular &amp;Velocity</string>
     </property>
     <property name="buddy">
      <cstring>gyroButton</cstring>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="gyroFileEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="gyroButton">
       <property name="text">
        <string>Select a file</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>&amp;Magnetic Field</string>
     </property>
     <property name="buddy">
      <cstring>magButton</cstring>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLineEdit" name="magFileEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="magButton">
       <property name="text">
        <string>Select a file</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>&amp;Pressure</string>
     </property>
     <property name="buddy">
      <cstring>pressureButton</cstring>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLineEdit" name="pressureFileEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="pressureButton">
       <property name="text">
        <string>Select a file</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>&amp;Temperature</string>
     </property>
     <property name="buddy">
      <cstring>tempButton</cstring>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLineEdit" name="tempFileEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="tempButton">
       <property name="text">
        <string>Select a file</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Analyzed &amp;data ( Position and Velocity )</string>
     </property>
     <property name="buddy">
      <cstring>analyzedButton</cstring>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLineEdit" name="analyzedFileEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="analyzedButton">
       <property name="text">
        <string>Select a file</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>1</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>accFileEdit</tabstop>
  <tabstop>accButton</tabstop>
  <tabstop>gyroFileEdit</tabstop>
  <tabstop>gyroButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>OpenFileDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>304</x>
     <y>400</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>188</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>OpenFileDialog</receiver>
   <slot>okClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>367</x>
     <y>400</y>
    </hint>
    <hint type="destinationlabel">
     <x>368</x>
     <y>184</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>okClicked()</slot>
 </slots>
</ui>
